spec/statement.dd

```diff
@@ -1176,6 +1176,29 @@ $(CONSOLE
 7.4 has type double
 hi has type string
 [2, 5] has type int[]
+)
+
+        $(P Each $(GLINK ForeachType) declaration will be a runtime variable,
+        unless `alias` or `enum` are specified:)
+
+        * The `alias` storage class will create a symbolic alias
+        * The `enum` storage class will create a compile-time constant
+
+$(SPEC_RUNNABLE_EXAMPLE_RUN
+---
+void main()
+{
+    import std.meta : AliasSeq;
+
+    int i = 1;
+
+    foreach (alias a; AliasSeq!(i, i))
+    {
+        a++;
+    }
+    assert(i == 3);
```

Comment:
The example is weird. I’d rather not use the same variable twice in the sequence.
```suggestion
    int i = 1, j = 2;

    foreach (alias a; AliasSeq!(i, j))
    {
        a++;
    }
    assert(i == 2 && j == 3);
```